PowerPointLabs/PowerPointLabs/ELearningLab/ELearningWorkspace/Model/ClickItem/ExplanationItem.cs

```diff
@@ -113,6 +114,8 @@ public string VoiceLabel
             {
                 voiceLabel = value;
                 NotifyPropertyChanged("VoiceLabel");
+                NotifyPropertyChanged("IsTriggerTypeComboBoxEnabled");
```

Comment:
Should the strings here be refactored into the constants file for elearningLab?

Reply:
Thanks for the suggestion! Refactored!